### DISCUSSION TITLE
Is the prompting functionality available in the latest Tabby release?

### DISCUSSION THREAD

**Discussion by factor3** [TabbyML/tabby#2381] [General]
I am running the latest docker version of Tabby, and am interested in using the chat playground described in your "what is Tabby" video (https://tabby.tabbyml.com/docs/welcome/). According to your documentation, in order to enable the playground in the docker container, I had to start the container setting the model to one of the chat models listed in your Model Registry (https://tabby.tabbyml.com/docs/models/). The model I am using is the Mistral model (TabbyML/Mistral-7B).

Unfortunately, the link to the Playground does not appear on the main page. I do not know why. Could it  be that the UI offered by the Tabby docker container has not been upgraded to include the Playground link? Or perhaps the container is a version behind the other installations?

Or maybe there is something in the configuration I need to do in order to make the playground work? Something not documented?

Someone please advise.

**Comment by wsxiaoys** [comment]
Hi, could you share the command you used to start tabby? to enable chat playground, it should be something like `--chat-model Mistral-7B`

**Reply by factor3** [reply]
No problem:

I ran tabby using docker compose. The compose file is below:

    version: '3.5'

>     services:
>      tabby:
>        restart: always
>        container_name: tabby
>        image: tabbyml/tabby
>        command: serve --model TabbyML/Mistral-7B --device cuda
>        volumes:
>          - "$HOME/tabby/data:/data"
>        ports:
>          - 8080:8080
>        deploy:
>          resources:
>            reservations:
>              devices:
>                - driver: nvidia
>                  count: 1
>                  capabilities: [gpu]

**Reply by wsxiaoys** [reply]
You need to specify it as below
```
command: serve --chat-model TabbyML/Mistral-7B --device cuda
```

**Reply by factor3** [reply]
Aaaaagh! My problem was due to a typo!

Thanks, wsxaioys. Making that change activated the playground. I feel like a dunce, but I am good to go.
